presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -273,6 +272,7 @@ primaryExpression
     | identifier '->' expression                                                     #lambda
     | '(' identifier (',' identifier)* ')' '->' expression                           #lambda
     | '(' query ')'                                                                  #subqueryExpression
+    | EXISTS '(' query ')'                                                           #exists
```

Comment:
Interesting. If I remember correctly, that's not valid ANSI SQL, but let me take a closer look at the spec.